mechanics/CREATING-A-SANDBOX-REPO.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Creating a sandbox repo"
+linkTitle: "Sandbox repo process"
+weight: 30
+type: "docs"
+---
+
+# Background
+
+The [`knative-sandbox`](https://github.com/knative-sandbox) GitHub org exists to
+hold "non-core" Knative components which are owned and sponsored by a Working
+Group but are not part of an official Knative release.
```

Comment:
Is this strictly true? @n3wscott is looking at moving camel & couchdb sources into sandbox (due to dep issues) and both of those were part of the release assests for eventing-contrib

Reply:
I think @n3wscott's plan is currently a tactical maneuver, but it looks like [there is also a proposal to migrate most of these repos](https://docs.google.com/document/d/1UcMNGKTjKdxLY_IVI7Xv5ZSOXAiO9DUrcuw2gkt74qY/edit#heading=h.lk82as48r7r3), so either couchdb/camel-k will move back to the `knative` repo, or all the other contents will be moved to `-sandbox` as well.

Reply:
right -- and it's if the latter case occurs I'm wondering if that line is strictly true or not